src/read_next/read_next.jsx

```diff
@@ -1,8 +1,11 @@
-import React, { useEffect, useState } from "react";
-import "./read_next.css";
+import React, { useEffect, useState } from 'react';
+import Pagination from "https://cdn.skypack.dev/rc-pagination@3.1.15";
```

Comment:
if you want to use 3rd party libraries, then install them with your package manager. if you use `npm` then `npm install`